server/src/main/java/org/elasticsearch/index/mapper/BooleanFieldMapper.java

```diff
@@ -208,14 +242,21 @@ public Query rangeQuery(Object lowerTerm, Object upperTerm, boolean includeLower
     private final boolean indexed;
     private final boolean hasDocValues;
     private final boolean stored;
+    private final Script script;
+    private final FieldValues<Boolean> scriptValues;
+    private final ScriptCompiler scriptCompiler;
```

Comment:
One feature freeze is gone I want to look seriously at moving `merge` to Builder objects, having to carry this stuff around is a real pain.

Reply:
agreed! I am happy to help with that